File: Harvard cs50 course/Credit/cash.py
from cs50 import get_float
while True:
    a = get_float("Change owed: ")
    if a >= 0.0:
        break

b = round(a * 100)
sum = 0
sum += int((b / 25)) + int(((b % 25) / 10)) + int((((b % 25) % 10) / 5)) + int((((b % 25) % 10) % 5))
print(sum)

File: Harvard cs50 course/Simple Cash/cash.c
#include<cs50.h>
#include<stdio.h>
#include<math.h>

int main(void)
{
    float a;
    do
    {
        a = get_float("Change owed: ");
    }
    while (a < 0.0);  //while loop
    int b = round(a * 100);
    int sum = 0;
    sum += (b / 25) + ((b % 25) / 10) + (((b % 25) % 10) / 5) + (((b % 25) % 10) % 5); // adding coins
    printf("%d\n", sum); //printing
}
